nilearn/plotting/displays.py

```diff
@@ -206,8 +206,9 @@ class GlassBrainAxes(BaseAxes):
     volumes with a schematic view of the brain.
 
     """
-    def __init__(self, ax, direction, coord, **kwargs):
+    def __init__(self, ax, direction, coord, plot_negative=False, **kwargs):
```

Comment:
I agree with this. It was the only comment I would have put, for readability, but the current state is fine with me as well, because sufficiently explained in the docstring.